package.json

```diff
@@ -12,8 +12,8 @@
     "prestart": "run-p setup clean db:dev",
     "init": "run-p db",
     "start": "run-p start:main start:worker",
-    "start:main": "tsnd --watch=.env,db/config.json --inspect=0.0.0.0:9229 --respawn --transpile-only -- src/main.ts",
-    "start:worker": "tsnd --watch=.env,db/config.json --inspect=0.0.0.0:9230 --respawn --transpile-only -- src/worker.ts",
+    "start:main": "tsnd --watch=.env,db/config.json,**/*.hbs --inspect=0.0.0.0:9229 --respawn --transpile-only -- src/main.ts",
+    "start:worker": "tsnd --watch=.env,db/config.json,**/*.hbs --inspect=0.0.0.0:9230 --respawn --transpile-only -- src/worker.ts",
```

Comment:
Fixing dev server to restart when changing handlebar files so that they show the latest content on refresh.